README.md

```diff
@@ -20,19 +24,14 @@ And then execute:
 
 Or install it yourself as:
 
-    $ gem install rdf-ldp
+    $ gem install rdf-vocab  # (was rdf-ldp)
```

Comment:
(Somehow that comment above got attached to the wrong lines on the README.  Likely because my HTML page became out-dated while we both worked on it and I didn't see the notification to re-sync until after making the comments.)